### ISSUE TITLE
Vignettes without `Examples`

### ISSUE THREAD

**Issue by konstantinakoukR** [pharmaverse/admiral#904]
The following vignettes don't contain `Examples` section:

- Add By Groups to all Datasets if Necessary
Source: [R/derive_param_tte.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_param_tte.R)

- Select the First or Last Date from Several Sources
Source: [R/derive_param_tte.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_param_tte.R)

- Default Format for Disposition Status
Source: [R/derive_var_disposition_status.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_var_disposition_status.R)

- Default format for the disposition reason
Source: [R/derive_vars_disposition_reason.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_vars_disposition_reason.R)

- Create an dthcaus_source object
Source: [R/derive_var_dthcaus.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_var_dthcaus.R)

- Create an lstalvdt_source object
Source: [R/derive_var_lstalvdt.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_var_lstalvdt.R)

- Pre-Defined Time-to-Event Source Objects
Source: [R/tte_sources.R](https://github.com/pharmaverse/admiral/blob/main/R/tte_sources.R)

- Default Format for Disposition Status
Source: [R/derive_var_disposition_status.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_var_disposition_status.R)

- Default format for the disposition reason
Source: [R/derive_vars_disposition_reason.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_vars_disposition_reason.R)

- Default Format for Disposition Status
Source: [R/derive_var_disposition_status.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_var_disposition_status.R)

- Default format for the disposition reason
Source: [R/derive_vars_disposition_reason.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_vars_disposition_reason.R)

- Deprecated - Add an Aggregated Parameter and Derive the Associated Start and End Dates
Source: [R/derive_param_exposure.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_param_exposure.R)

- Deprecated - Derive Baseline
Source: [R/derive_var_base.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_var_base.R)

- Deprecated - Derive BASEC Variable
Source: [R/derive_var_base.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_var_base.R)

- Deprecated - Add an Aggregated Parameter and Derive the Associated Start and End Dates
Source: [R/derive_param_exposure.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_param_exposure.R)

- Deprecated - Derive Baseline
Source: [R/derive_var_base.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_var_base.R)

- Deprecated - Add an Aggregated Parameter and Derive the Associated Start and End Dates
Source: [R/derive_param_exposure.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_param_exposure.R)

- Deprecated - Derive BASEC Variable
Source: [R/derive_var_base.R](https://github.com/pharmaverse/admiral/blob/main/R/derive_var_base.R)

**Comment by konstantinakoukR** [comment]
@thomas-neitmann regarding this one, after discussion with @bms63 maybe that this could be fixed in another release. What do you think?

**Comment by bms63** [comment]
Hey @konstantinakoukR are you rendering the devel version of the website?  I see examples for 
[R/derive_param_tte.R](https://github.com/Roche-GSK/admiral/blob/main/R/derive_param_tte.R).  Actually I see examples on 0.5.0 website, so maybe we need to clarify this issue a bit more as I'm confused

**Comment by bms63** [comment]
Oh I see.  The helper functions for derive_param_tte are missing examples.  Could you identify which ones in each `.R` are missing examples?  This would help someone picking up this issue to quickly come up to speed.

**Comment by rossfarrugia** [comment]
agree! @konstantinakoukR can you take a look and update this issue accordingly please? no updates needed for deprecated functions, but for the others if you clean up this issue then maybe could even add "good first issue" label for a new contributor to possibly pick up.

**Comment by bms63** [comment]
All of these have update examples or have been removed from admiral.  Going to close out.
